website/docs/reference/commands/test.md

```diff
@@ -28,4 +29,48 @@ dbt test --select "one_specific_model,test_type:singular"
 dbt test --select "one_specific_model,test_type:generic"
 ```
 
-For more information on writing tests, see the [Testing Documentation](/docs/build/data-tests).
+For more information on writing tests, see the [Testing Documentation](/docs/build/tests).
+
+</VersionBlock>
+
+<VersionBlock firstVersion="1.8">
+
+`dbt test` runs data tests and unit tests defined on models, sources, snapshots, and seeds. It expects that you have already created those resources through the appropriate commands.
```

Comment:
Potentially misleading cause you can define data tests on models, sources, snapshots, and seeds but you can currently only define unit tests on models

Reply:
Clarified it!